docs/openapi/paths/activity-reports/activity-recipients.yaml

```diff
@@ -14,6 +14,11 @@ get:
             type: object
             properties:
               grants:
+<<<<<<< HEAD
```

Comment:
And another merge conflict here

Reply:
Whoops, good catch. Should be fixed now.